lib/all-badge-examples.js

```diff
@@ -0,0 +1,2035 @@
+'use strict';
+
+const allBadgeExamples = [
+  {
+    sectionId: 'build',
+    sectionName: 'Build',
+    badges: [
+      {
+        title: 'Travis',
+        previewBadgeUri: '/travis/rust-lang/rust.svg',
+        exampleBadgeUri: 'https://img.shields.io/travis/USER/REPO.svg'
+      },
+      {
```

Comment:
Maybe we should add "category" too (ie. the heading it's currently under on try.html)? We could likely extract that from the current try.html too.

Reply:
Yea, for sure, it's in there now. They're called "sections" in this version.

When we break these out by service we can invert that, labeling each one with its category.